doc/tutorials/application-integration.md

```diff
@@ -205,39 +200,39 @@ For more information about that, continue reading
 
 ## Specification ##
 
-Now, we have a fully working configuration system without any hard-coded
-information (such as configuration files). So we already gained something.
+Now, we have a fully working configuration system without any hardcoded
```

Comment:
Maybe we should write a script that automatically does the replacements to a canonical form. Discussions for that in #920